packages/selinux-policy/rules.cil

```diff
@@ -179,12 +179,16 @@
 (allow all_s ephemeral_o (files (mutate mount)))
 
 ; Trusted subjects are allowed to write to and manage mounts
-; for "dynamic" files in /etc.
-(allow trusted_s dynamic_o (files (mutate mount)))
+; for files in /etc.
+(allow trusted_s etc_t (files (mutate mount)))
```

Comment:
`binfmt_misc_t` is now part of `dynamic_o`, so I made this rule more specific since the trusted software has no reason to define binfmt handlers.